reader/components/outline-sidebar/OutlineSidebar.jsx

```diff
@@ -46,11 +46,20 @@ class OutlineSidebar extends React.PureComponent {
     ]
 
     return (
-      <OutlineGroup
-        isExpanded
-        linkService={linkService}
-        outline={finalOutline}
-      />
+      <>
+        {!outline && (
+          <div className="missing-outline">
+            Document outline is not available for this moment. We are working
+            hard to bring this feature to bring this feature to majority of
+            works as soon as possible.
```

Comment:
Let's keep only this:
> Document outline is not available for this moment.

Reply:
https://github.com/oacore/reader/pull/106/commits/a8d706477d649c2151095655eeaaded63666f0a7